docs/migrating-to-velero.md

```diff
@@ -0,0 +1,81 @@
+# Migrating from Heptio Ark to Velero
+
+As of v0.11.0, Heptio Ark has become Velero. This means the following changes have been made:
+
+* The `ark` CLI client is now `velero`.
+* The default Kubernetes namespace and ServiceAccount are now named `velero` (formerly `heptio-ark`).
+* The container image name is now `gcr.io/heptio-images/velero` (formerly `gcr.io/heptio-images/ark`).
+* CRDs are now under the new `velero.io` API group name (formerly `ark.heptio.com`).
+
+
+The following instructions will help you migrate your existing Ark installation to Velero.
+
+# Prerequisites
+
+*  Ark v0.10.x installed. See the v0.10.x [upgrade instructions][1] to upgrade from older versions.
+* `kubectl` installed.
+* `cluster-admin` permissions.
+
+# Migration process
+
+At a high level, the migration process involves the following steps:
+
+* Scale down the `ark` deployment, so it will not process schedules, backups, or restores during the migration period.
+* Create a new namespace (named `velero` by default).
+* Apply the new CRDs.
+* Migrate existing Ark CRD objects, labels, and annotations to the new Velero equivalents.
+* Recreate the existing cloud credentials secret(s) in the velero namespace.
+* Apply the updated Kubernetes deployment and daemonset (for restic support) to use the new container images and namespace.
+* Remove the existing Ark namespace (which includes the deployment), CRDs, and ClusterRoleBinding.
+
+These steps are provided in a script here:
+
+```bash
+kubectl scale --namespace heptio-ark deployment/ark --replicas 0
+
+# Download the velero client/example tarball to $VELEROROOT and unpack
```

Comment:
I think we can remove references to `$VELEROROOT` since everything needed will be downloaded/unpacking in pwd anyway - right?